jsk_denso_robot/sample_pack/canon_camera/src/orinexception.py

```diff
@@ -0,0 +1,78 @@
+﻿'''
+Software License Agreement (MIT License)

```

Comment:
where these codes comes from? please create canon_camera/denso? /bcap? direcotry and put 3rd-party cond within there.